toolkit/tools/pkg/imagecustomizerlib/shrinkfilesystems.go

```diff
@@ -85,7 +85,8 @@ func shrinkFilesystems(imageLoopDevice string, verityHashPartition *imagecustomi
 		}
 
 		// Shrink the file system with resize2fs -M
-		stdout, stderr, err := shell.Execute("resize2fs", "-M", partitionLoopDevice)
+		stdout, stderr, err := shell.Execute("flock", "--timeout", "5", imageLoopDevice,
```

Comment:
For consistency maybe use the same `timeoutInSeconds = "5"` pattern (or again, maybe this just lives in a dedicated spot).

Reply:
The Image Customizer codebase doesn't use that code pattern as much. Hence the inconsistency.